velox/functions/prestosql/aggregates/MinMaxByAggregates.cpp

```diff
@@ -1100,6 +1114,16 @@ template <
     template <typename U, typename V>
     class NAggregate>
 exec::AggregateRegistrationResult registerMinMaxBy(const std::string& name) {
+  std::vector<std::shared_ptr<exec::AggregateFunctionSignature>> signatures;
+  // V, C -> row(V, C) -> V.
+  signatures.push_back(exec::AggregateFunctionSignatureBuilder()
```

Comment:
The PR title doesn't seem to match the code. The title says "fix nested nulls", but the actual changes seems to "Add support for complex types to compare argument of min_by and max_by".

Reply:
>actual changes seems to "Add support for complex types to compare argument of min_by and max_by".

Yes, we do not support complex type compare arguments before so there are no *nested nulls issues*. This PR actually enables the support along with the right way to handle nested nulls. I will update the RP title.